io_export_cryblend/utils.py

```diff
@@ -933,6 +933,9 @@ def apply_animation_scale(armature):
 
 def get_bone_geometry(bone_name):
     try:
+        if bone_name.endswith("_Phys"):
+            bone_name = bone_name[:-5]

```

Comment:
I get that error for del operation,

TypeError: 'str' object does not support item deletion

It seems str does not support del operation.